Goobi/src/de/sub/goobi/beans/Prozess.java

```diff
@@ -487,7 +487,7 @@ public boolean accept(File dir, String name) {
 			};
 
 			String origOrdner = "";
-			String[] verzeichnisse = dir.list(filterVerz);
+			String[] verzeichnisse = FilesystemHelper.list(dir, filterVerz);
 			for (int i = 0; i < verzeichnisse.length; i++) {
 				origOrdner = verzeichnisse[i];
```

Comment:
Could be simplified to `origOrdner = verzeichnisse[verzeichnisse.length];` but that is a different pull request then.